[DO NOT MERGE][Draft] New version 2021-10-01

zhaomuzhi opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit (including refactoring) and then push new changes, including version updates, in separate commits.

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

zhaomuzhi committed:
Merge remote-tracking branch 'upstream/master'

zhaomuzhi committed:
change for getTransitivePEUsages

zhaomuzhi committed:
change for getTransitivePEUsages

zhaomuzhi committed:
update

zhaomuzhi committed:
update

zhaomuzhi committed:
prettier code

zhaomuzhi committed:
prettier code

zhaomuzhi committed:
add another new endpoint

zhaomuzhi committed:
add another new endpoint

zhaomuzhi committed:
address comments

zhaomuzhi committed:
address comments

zhaomuzhi committed:
revert change on transitivePE and notebookKeys

zhaomuzhi committed:
Merge branch 'master' of https://github.com/zhaomuzhi/azure-rest-api-specs

zhaomuzhi committed:
revert change on transitivePE and notebookKeys

zhaomuzhi committed:
Merge remote-tracking branch 'upstream/master'

zhaomuzhi committed:
Merge remote-tracking branch 'upstream/master'

zhaomuzhi committed:
add new stable version 2021-01-01

zhaomuzhi committed:
add ListNotebookAccessToken

zhaomuzhi committed:
add primaryUserAssignedIdentity

zhaomuzhi committed:
fix missing properties and apis

zhaomuzhi committed:
fix one more missing property

zhaomuzhi committed:
Merge remote-tracking branch 'upstream/master'

VinayRSoni committed:
202 status for start and stop CI

zhaomuzhi committed:
Merge pull request #8 from zhaomuzhi/vison/202startstopCI

202 status for start and stop CI

zhaomuzhi committed:
Merge remote-tracking branch 'upstream/master'

SubaMoorthy committed:
make resync keys long runnin operation in version Jan2021

SubaMoorthy committed:
resync keys 204

SubaMoorthy committed:
update resync keys example

SubaMoorthy committed:
Dont throw 204 for resync

SubaMoorthy committed:
update resync keys

SubaMoorthy committed:
Formatting

SubaMoorthy committed:
revert Jan2021changes and add new version

SubaMoorthy committed:
start apr2021 version for azure ml rp

SubaMoorthy committed:
undo remove new line in jan2021

zhaomuzhi committed:
Merge branch 'master' of https://github.com/SubaMoorthy/sukaruna-aml-apr2021

zhaomuzhi committed:
use systemData definition in common

zhaomuzhi committed:
removed old files

zhaomuzhi committed:
Merge pull request #3 from SubaMoorthy/referenceCommonSystemData

Reference common system data

SubaMoorthy committed:
fix small swagger bugs

SubaMoorthy committed:
Merge pull request #4 from SubaMoorthy/fix-swagger-kpi-bugs

fix small swagger bugs

SubaMoorthy committed:
run prettier

SubaMoorthy committed:
Merge pull request #6 from SubaMoorthy/fix-swagger-kpi-bugs

run prettier

SubaMoorthy committed:
remove reference to sku

SubaMoorthy committed:
Merge pull request #7 from SubaMoorthy/fix-swagger-kpi-bugs

remove reference to sku

SubaMoorthy committed:
update readme

SubaMoorthy committed:
Merge pull request #8 from SubaMoorthy/fix-swagger-kpi-bugs

update readme

Commit:
minimum agentCount should be 0

Commit:
add createdOn and ModifiedOn to compute properties

Commit:
add LoadBalancerType to aks compute

Commit:
add isNotebookInstanceCompute to virtualMachine

Commit:
add leafDomainLabel to SslConfiguration

Commit:
add overwriteExistingDomain to SslConfiguration

Commit:
add synapseSpark as supported compute-type

Commit:
update patch compute response to 202

Commit:
remove location from example responses

Commit:
Revert "remove location from example responses"

This reverts commit 048bf2551513afae842a9c4fbb5861aeddc6c33a.

joshanator committed:
Merge pull request #9 from SubaMoorthy/jopet/mlc-swagger-fixes

MLC swagger fixes

Commit:
MachineLearningCompute_Update should have a 200 response

SubaMoorthy committed:
sys data to use local def due to naming conflicts.

SubaMoorthy committed:
Merge pull request #10 from SubaMoorthy/fix-swagger-kpi-bugs

sys data to use local def due to naming conflicts.

SubaMoorthy committed:
Fix enum casing

SubaMoorthy committed:
Merge pull request #11 from SubaMoorthy/fix-swagger-kpi-bugs

Fix enum casing

SubaMoorthy committed:
Fix operationId PR comment.

SubaMoorthy committed:
Merge pull request #12 from SubaMoorthy/fix-swagger-kpi-bugs

Fix operationId PR comment.

SubaMoorthy committed:
Change opId in apr version

SubaMoorthy committed:
Merge pull request #13 from SubaMoorthy/fix-swagger-kpi-bugs

Change opId in apr version

SubaMoorthy committed:
remove 20210401 for easy review

SubaMoorthy committed:
only change api version

SubaMoorthy committed:
merge changes for 20210401 version

Commit:
Add x-ms-discriminator-value for Synapse spark

Commit:
camelcase loadbalancer properties

Commit:
add createdOn and Modified on to examples

Commit:
address AvoidAnonymousTypes error for SynapseSpark

SubaMoorthy committed:
Minor fixes + new property

SubaMoorthy committed:
Remove subscription state.

SubaMoorthy committed:
Merge branch 'Azure:master' into master

SubaMoorthy committed:
Merge branch 'Azure:master' into master

zhaomuzhi committed:
Merge branch 'master' of https://github.com/Azure/azure-rest-api-specs

zhaomuzhi committed:
Merge branch 'master' of https://github.com/Azure/azure-rest-api-specs into newVersion20210701_locked

zhaomuzhi committed:
copy over from baseline api version 2021-03-01-preview

zhaomuzhi committed:
removed preview features from 2021-03-01-preview and added more apis and properties in 2021-07-01

zhaomuzhi committed:
remove more preview features and examples

zhaomuzhi committed:
delete more examples of preview features

zhaomuzhi committed:
do some fixes

zhaomuzhi committed:
add a new enum value in DiagnoseResult

zhaomuzhi committed:
fix bug

zhaomuzhi committed:
update example

RichardZhaoW committed:
Merge branch 'Azure:master' into newVersion20210701_locked

Commit:
add x-ms-secret to some fields of KubernetesProperties

zhaomuzhi committed:
Merge pull request #23 from RichardZhaoW/newVersion20210701_locked

add x-ms-secret to some fields of KubernetesProperties

zhaomuzhi committed:
fix example

zhaomuzhi committed:
Merge branch 'newVersion20210701_locked' of https://github.com/zhaomuzhi/azure-rest-api-specs into newVersion20210701_locked

zhaomuzhi committed:
fix build error

zhaomuzhi committed:
add one new diagnosis category

zhaomuzhi committed:
fix swagger example

zhaomuzhi committed:
address comments on ListNodes api

zhaomuzhi committed:
move 2021-07-01 version to 2021-10-01

zhaomuzhi committed:
update api version in multi places and updated the ListSkus api to make it aligned with service

zhaomuzhi committed:
Merge branch 'master' of https://github.com/Azure/azure-rest-api-specs into newVersion20211001

zhaomuzhi committed:
fix build error

zhaomuzhi committed:
remove new versions for go and java sdk for now

forteddyt committed:
Merge branch 'main' of github.com:Azure/azure-rest-api-specs

forteddyt committed:
Merge branch 'main' into newVersion20211001

forteddyt committed:
initial 2021-10-01 mfe.json + readme update

forteddyt committed:
migrate mls.json to common-types sku

forteddyt committed:
externalize mfe to common-types sku

forteddyt committed:
temp rename of definitions to side-step name collisions

forteddyt committed:
update PartialOnlineDeploymentPartialTrackedResource to use partials

forteddyt committed:
mark mfe props w/ x-nullable

forteddyt committed:
bump example api-versions to 2021-10-01

forteddyt committed:
update BatchEndpoint examples

forteddyt committed:
update BatchDeployment examples

forteddyt committed:
update CodeVersion examples

forteddyt committed:
update ComponentVersion examples

forteddyt committed:
update DatasetVersion examples

forteddyt committed:
remove Azure SQL Datastore examples

forteddyt committed:
update Datastore examples

forteddyt committed:
update EnvironmentVersion examples

forteddyt committed:
update Job examples

forteddyt committed:
update ModelVersion examples

forteddyt committed:
update OnlineDeployment examples

forteddyt committed:
update OnlineEndpoint examples

forteddyt committed:
remove unused example files

forteddyt committed:
prettier

forteddyt committed:
remove identity until Azure/autorest#4262 is addressed

forteddyt committed:
add trainingjob to custom-words

forteddyt committed:
add temp TODO to descriptions for R3011

forteddyt committed:
processsCountPerInstance -> processCountPerInstance

forteddyt committed:
defintion -> definition

forteddyt committed:
typo fix in BatchDeployment examples

forteddyt commented:
Regarding [LintDiff R3011 failures](https://github.com/Azure/azure-rest-api-specs/pull/15439/checks?check_run_id=3341238861), I've opened https://github.com/Azure/azure-rest-api-specs/issues/15663. It looks like R3011 is being applied to example json files, which seems like a bug on the LintDiff. See Issue for more details

forteddyt commented:
Regarding the removal of Identity from common-types' ResourceModelWithAllowedPropertySet, I've opened https://github.com/Azure/autorest/issues/4262. This seems like an AutoRest core version 2.x issue. The removal **is temporary** to unblock gates and check for other spec issues. See Issue for more details

forteddyt committed:
remove workspaceSkus.json

forteddyt committed:
transfer ownership of ContainerResourceRequirements to mfe

forteddyt committed:
remove unused ServicePrincipalCredentials definition

forteddyt committed:
implement fix as suggested by Azure/autorest#4262

forteddyt committed:
reorder and redefine ContainerResourceRequirements

forteddyt committed:
Revert "remove identity until Azure/autorest#4262 is addressed"

This reverts commit 9fe04ab84eab6c1a0d788f7365cd2944a2ad219c.

forteddyt committed:
change various Sku definition names back, now that workspaceSkus.json is removed

forteddyt commented:
Might as well file an issue for R2008 as well. This was originally noticed in 2021-03-01-preview, but it's still around so... https://github.com/Azure/azure-rest-api-specs/issues/15666. See Issue for more details

comacgin committed:
add compute propertyBag

comacgin committed:
Explicit additional properties type is required

comacgin committed:
empty schema

comacgin committed:
move propertybag from compute to amlcompute only

forteddyt commented on specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json:
```diff
@@ -0,0 +1,10010 @@
+        },
+        "properties": {
+          "description": "The asset property dictionary.",
+          "t
```
It's my understanding that "name" is a reserved word and cannot be used as a property name (see https://github.com/Azure/azure-rest-api-specs/pull/12554#issuecomment-812245037). @pilor is SkuSetting.name allowed?

pilor replied:
SKUs have a standard contract.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#enumerating-skus-for-an-existing-resource

forteddyt commented on specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json:
```diff
@@ -0,0 +1,10010 @@
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/onlineEndpoints/{endpointName}/deployments/{deploymentName}/skus": {
+      "get": {
```
For some reason, I have it in my head that all GET requests MUST respond with a Resource-type object. Is this true? Because GET /skus responds with [DeploymentSkuResourceTypeArmPaginatedResult](https://github.com/Azure/azure-rest-api-specs/blob/326e72400058709b0b15eb7ed4b3a02ee6816498/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json#L6906), which is not an x-ms-azure-resource. @pilor can you double check?

pilor replied:
In general yes that is true.  Skus have a lot of legacy and don't follow that pattern.  See: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#enumerating-skus-for-an-existing-resource

forteddyt committed:
update SkuTier to match common-types definition

forteddyt committed:
move to verbatim definition of Sku, as suggested by Azure/autorest#4262

forteddyt committed:
migrate mls to use mfe.json Sku (which is verbatim copy of common-types)

forteddyt committed:
Revert "implement fix as suggested by Azure/autorest#4262"

This reverts commit ced11a7f0f67b1c7c568b2f4dff277b95616eb72.

forteddyt committed:
add + reference types' Sku/SkuTier

forteddyt committed:
add missing required properties 'path' to types.json

forteddyt committed:
add required securityDefinitions

forteddyt committed:
update mfe.json with latest changes

forteddyt committed:
update examples

forteddyt committed:
prettier

forteddyt committed:
set requestSettings default to {}

forteddyt committed:
Revert "set requestSettings default to {}"

This reverts commit ffc00154a8b89c991b81d5a24c237eab6a6b44e3.

forteddyt committed:
move readinessProbe property to OnlineDeployment

forteddyt committed:
match Partial's x-nullability with their non-Partial equivalent

forteddyt committed:
remove x-nullable from identity, kind, sku tracked resource props

forteddyt committed:
add case-insensitive comments

forteddyt committed:
remove containerName

j5lim commented:
>         "accessToken": {

Can you annotate it with x-ms-secret?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json:7167 in 0be44c7. [](commit_id = 0be44c73188565bc812698c92d312e016e9b72aa, deletion_comment = False)

j5lim reviewed (approved):
Approved from ARM side.

forteddyt committed:
remove containerName from examples

forteddyt committed:
update w/ more defaults

forteddyt committed:
K8s -> Kubernetes

forteddyt committed:
accessToken->secrets secret

forteddyt committed:
add missing example files

forteddyt committed:
Revert "accessToken->secrets secret"

This reverts commit dfceddd834ef9b881095d8500be14dd11ac2d794.

forteddyt commented:
> > ```
> >     "accessToken": {
> > ```
> 
> Can you annotate it with x-ms-secret?
> 
> Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json:7167 in [0be44c7](https://github.com/Azure/azure-rest-api-specs/commit/0be44c73188565bc812698c92d312e016e9b72aa). [](commit_id = [0be44c7](https://github.com/Azure/azure-rest-api-specs/commit/0be44c73188565bc812698c92d312e016e9b72aa), deletion_comment = False)

From email conversation with another ARM member a few months ago:
> You must use different definitions, one where secrets are annotated (this definition is shared across PUT, PATCH, GET), and another definition for POST where you would not annotate secrets.

Since accessToken is part of a schema that is only used in POST responses, accessToken should not be annotated with "x-ms-secret".

Commit:
Add Byo Cmk support

Commit:
Space

Commit:
Space

zhaomuzhi committed:
Merge pull request #26 from superkevin813/byocmksupport

Add Byo Cmk support

j5lim commented:
> > > ```
> > >     "accessToken": {
> > > ```
> > 
> > 
> > Can you annotate it with x-ms-secret?
> > Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json:7167 in [0be44c7](https://github.com/Azure/azure-rest-api-specs/commit/0be44c73188565bc812698c92d312e016e9b72aa). [](commit_id = [0be44c7](https://github.com/Azure/azure-rest-api-specs/commit/0be44c73188565bc812698c92d312e016e9b72aa), deletion_comment = False)
> 
> From my email conversation with another ARM member a few months ago:
> 
> > You must use different definitions, one where secrets are annotated (this definition is shared across PUT, PATCH, GET), and another definition for POST where you would not annotate secrets.
> 
> Since accessToken is part of a schema that is only used in POST responses, accessToken should not be annotated with "x-ms-secret".

Please ignore my comment then. You don't need to annotate the property with x-ms-secret if it is only used by POST.

Commit:
Fix upper case

zhaomuzhi committed:
Merge pull request #27 from superkevin813/byocmksupport

Fix upper case

zhaomuzhi committed:
revert BYO cmk resources feature

forteddyt committed:
update SearchSpace to just object type

forteddyt committed:
update CommandJob.Parameters to just object type

forteddyt committed:
ake JobLimits polymorphic

forteddyt committed:
restructure spec w/ hierarchical relationships

forteddyt committed:
add more defaults

forteddyt committed:
fix container resource requirement examples

forteddyt committed:
add KeyType example

erjms committed:
Updated mfe spec

- Updated ModelVersion contract to make modelUri optional

forteddyt committed:
bump mfe to common-types v3

forteddyt committed:
bump mls to common-types v3

forteddyt committed:
bump workspaceFeatures to common-types v3

forteddyt committed:
Merge branch 'main' of github.com:Azure/azure-rest-api-specs

forteddyt committed:
Merge branch 'newVersion20211001' of github.com:zhaomuzhi/azure-rest-api-specs into newVersion20211001

forteddyt committed:
Merge branch 'main' into newVersion20211001

forteddyt committed:
use common-types v3 Sku and SkuTier

forteddyt committed:
remove types.json from readme.md

forteddyt committed:
rename conflicting ResourceIdentityType to TEMPORARY name

forteddyt committed:
rename conflicting Identity to TEMPORARY name

forteddyt committed:
revert bump to common-types v3

zhaomuzhi committed:
make patch workspace a long running operation

zhaomuzhi committed:
allow update applicationInsights and containerRegistry

forteddyt committed:
bump to v3

forteddyt committed:
update descriptions + add example

forteddyt committed:
add new Default

forteddyt committed:
thin out PartialOnlineDeployment

forteddyt committed:
update x-nullable for some properties

forteddyt committed:
rename mls resources to non-conflicting names

forteddyt committed:
move mls to mfe's ResourceIdentity dependency

forteddyt committed:
update examples with thinned PartialOnlineDeployment

harneetvirk commented:
/azp run

forteddyt committed:
decorate resource models with x-ms-client-name

forteddyt committed:
default instanceCount to 1 and ResourceConfiguration to default obj

forteddyt committed:
make environmentId and Trial required

forteddyt committed:
example modelValidation fix

forteddyt committed:
description updates

forteddyt committed:
rename /skus api models

libc16 committed:
add nullable for subnet of AmlComputeProperties

libc16 committed:
fix some missing type error of mlc

libc16 committed:
add nullable for subnet of AmlComputeProperties

zhaomuzhi committed:
Merge pull request #31 from libc16/fix_miss_type_mlc

[MLC] fix some missing type error of mlc

zhaomuzhi committed:
Merge pull request #30 from libc16/MLC_subnet_nullable

[MLC] add nullable for subnet of AmlComputeProperties

forteddyt committed:
accountName & containerName for AzureBlobDatastore no longer required

libc16 committed:
[MLC] Fix swagger Bugs/Errors of MLC

forteddyt committed:
Merge remote-tracking branch 'upstream/main' into newVersion20211001

zhaomuzhi committed:
Merge pull request #28 from ghost/newVersion20211001

add nullable for subnet of AmlComputeProperties

ZhidaLiu committed:
resolve SystemData error

adding ref to Resource to resolve the RequiredReadOnlySystemData error

ZhidaLiu committed:
Resolve missing default response

ZhidaLiu committed:
Resolve wrong response schema

ZhidaLiu committed:
Add missing Object type

libc16 committed:
Fix swagger lint issue in machineLearningService.json

ZhidaLiu committed:
Remove duplicate properties after using AllOf keyword

zhaomuzhi committed:
Merge pull request #32 from libc16/fix_mlc_swagger_bugs

[MLC] Fix swagger Bugs/Errors of MLC

forteddyt commented:
/azp run

forteddyt committed:
description change to trigger test rerun

ZhidaLiu committed:
Adding missing example and small fix

zhaomuzhi committed:
Merge pull request #33 from ZhidaLiu/zhili-swagger-error-fix

Swagger error fix for workspaceRP APIs

libc16 committed:
[MLC] fix ModelValidation Errors of MLC

zhaomuzhi committed:
Merge pull request #34 from libc16/fix_modelvalidation_mlc

[MLC] fix ModelValidation Errors of MachineLearningCompute

ZhidaLiu committed:
try to fix mutability error

ZhidaLiu committed:
fix mutability issue again

ZhidaLiu committed:
try fix mutability error

ZhidaLiu committed:
suppress wrong error for pageable itemName override

ZhidaLiu committed:
fix 3011 error

ZhidaLiu committed:
Revert "fix 3011 error"

This reverts commit 5464fc194b3740f671a259bc74aa357bf2078351.

ZhidaLiu committed:
Revert "try fix mutability error"

This reverts commit 52083e42107a50b9e4c1a1d9e5fb7ffd50bed9b0.

ZhidaLiu committed:
Revert "fix mutability issue again"

This reverts commit 86f79106953cd22673afb359482665d3faa54644.

ZhidaLiu committed:
Revert "try to fix mutability error"

This reverts commit d4aa7a04d1ddef525c3b49bc141b3959e44105dc.

zhaomuzhi committed:
Merge pull request #35 from ZhidaLiu/users/zhili/WRP_1001_SwaggerFix

Users/zhili/wrp 1001 swagger fix

ZhidaLiu committed:
move 1001 folder to machinelearning

ZhidaLiu committed:
remove old README and update suppression

ZhidaLiu committed:
update suppression

ZhidaLiu committed:
fix

ZhidaLiu committed:
fix

ZhidaLiu committed:
Merge pull request #36 from ZhidaLiu/user/zhili/FolderMove1001

Move 1001 folder to machinelearning

forteddyt committed:
remove allowPublicAccess property

forteddyt committed:
add ListViewType and isArchived

akning-ms commented:
/azp run

forteddyt committed:
fix stray /v2 common-types reference

forteddyt committed:
remove allowPublicAccess usage from examples

forteddyt committed:
typo fixes

forteddyt committed:
add autosave and unmount to custom-words

erjms committed:
Removed x-ms-long-running-operation-options from MFE spec

Removed from 2021-10-01 api version. Only removed for non-POST LRO operations.
Non-breaking change to fix autogenerated REST clients.

ArcturusZhang commented:
Any update on this?
